aioresponses/core.py

```diff
@@ -469,8 +469,12 @@ async def match(
             else:
                 return None
 
-            if matcher.repeat is False:
+            if matcher.repeat in (False, 1):
```

Comment:
test are not working and this part is bugged.
when repeat is set to True the if statement does not work [True in (False, 1) returns True].

IMO we should have
```python
if isinstance(matcher.repeat, bool):
    if not matcher.repeat:
        del self._matches[key]
else:
    if matcher.repeat == 1:
        del self._matches[key]
    matcher.repeat -= 1
```

Reply:
Great suggestion, much cleaner. I'll gladly fix it. Just give me a few days, please (got a project deadline).